pantheon-bundle/frontend/src/app/__snapshots__/searchBeta.test.tsx.snap

```diff
@@ -347,7 +352,7 @@ exports[`SearchBeta tests should render default Search component 1`] = `
         width="width_50"
       >
         <ExpandableSection
-          className="pf-c-title"
+          className="pf-c-title searh-results__section search-results__section--module"
```

Comment:
Misspelled the first `search` as `searh`, but otherwise it's great!

Then if we ever wanted a class on `<DrawerContentBody>` I'd make it `search-results` and this would be an A+ representation of BEM.